### PR TITLE
[#1505] Log byte arrays as Hex Strings

### PR DESCRIPTION
**Pull request by nyetwurk**
Make sure we don't log raw []byte arrays. Convert them to Hex Strings.

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by charleszqwang** [comment]
test this please

**Comment by charleszqwang** [comment]
Jenkins, test this please.

**Comment by charleszqwang** [comment]
test this please

**Comment by xChickens** [comment]
test this please

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Comment by xChickens** [comment]
Jenkins, test this please.

File: node/node_handler.go [hunk 1]
```diff
@@ -478,8 +478,8 @@ func (node *Node) AddNewBlock(newBlock *types.Block) error {
 		utils.Logger().Error().
 			Err(err).
 			Uint64("blockNum", newBlock.NumberU64()).
-			Bytes("parentHash", newBlock.Header().ParentHash.Bytes()[:]).
-			Bytes("hash", newBlock.Header().Hash().Bytes()[:]).
```
Comment (nyetwurk) [inline node/node_handler.go]: As a side node I think `Bytes()[:]` its the same as `Bytes()` since `Bytes()` implicitly uses `len()`

Comment (nyetwurk) [reply node/node_handler.go]:
```
// Bytes gets the byte representation of the underlying hash.
func (h Hash) Bytes() []byte { return h[:] }
```

**Pull request merged**